uniffi_docs/src/lib.rs

```diff
@@ -103,16 +193,9 @@ pub fn extract_documentation(path: &Utf8Path) -> Result<Documentation> {
                                     None
                                 }
                             })
+                            // TODO.Currently unwrap, as we do not expect errors. Just being compliant with API. Better handling.
```

Comment:
Ok. I agree, let's remove the comment.

Reply:
Fixed at e33d469eb595801c5477968d848a7a0ede9d9de0.